module/zfs/vdev_raidz.c

```diff
@@ -1436,33 +1323,40 @@ vdev_raidz_reconstruct(raidz_map_t *rm, int *t, int nt)
 	dt = &tgts[nbadparity];
 
 	/*
+	 * Reconstruct using the new math implementation if
+	 * rm_ops is set.
+	 */
+	if (rm->rm_ops) {
+		return (vdev_raidz_math_reconstruct(rm, parity_valid, dt,
+		    nbaddata));
+	}
+
+	/*
 	 * See if we can use any of our optimized reconstruction routines.
 	 */
-	if (!vdev_raidz_default_to_general) {
```

Comment:
It was a file-scope global, so compiler would init it to 0. It doesn't have to have that _static_.

On systems other than Linux, I believe there's mechanisms to change such variables at run-time without explicitly declaring them as parameters.

I'm not arguing whether to keep this change. I just wanted to understand why it was made, since the description of this patch didn't mention anything about it.